docs/src/appendix/cheatcodes/get_class_hash.md

```diff
@@ -0,0 +1,46 @@
+# `get_class_hash`
+
+> `fn get_class_hash(contract_address: ContractAddress) -> ClassHash`
+
+Retrieves a class hash of a contract deployed under the given address.
+
+- `contract_address` - target contract address
+
+The main purpose of this cheatcode is to test upgradable contracts. For contract implementation:
+
+```rust
+    // ...
+
+    #[external(v0)]
+    impl IUpgradeableImpl of super::IUpgradeable<ContractState> {
+        fn upgrade(ref self: ContractState, class_hash: starknet::ClassHash) {
+            starknet::replace_class_syscall(class_hash).unwrap_syscall();
+        }
+    }
+
+    // ...
+```
```

Comment:
I know I'm being picky, but can we get rid of the empty lines? xD

Reply:
Fine to be picky, thanks a lot for the feedback.